app/Http/Requests/SplitRequest.php

```diff
@@ -24,6 +24,7 @@ public function rules()
         return [
             'competition_end_date' => 'required|date|after:today',
             'leaderboard_msg_day' => 'required',
+            'rules' => 'required|url',
```

Comment:
You use `:rules_ur:` in the body fields of the default messages but didn't change it here.